src/app/component/places/places.component.html

```diff
@@ -0,0 +1,78 @@
+<main role="main" id="main-content">
+  <div class="main container">
+    <div class="places-filter">
+      <div class="search">
+        <div class="search-elements">
+          <img class="search-icon" src="assets/img/places/search.svg">
+          <input class="form-control search-write" type="text" name="search" list="places_names"
+            [(ngModel)]="searchName" placeholder="{{ 'user.map.searching-for-a-place' | translate }}" />
```

Comment:
maybe here would be better to add some binding with variable for atribute src like in other images in our app..?

Reply:
Done